.gitignore

```diff
@@ -62,3 +62,7 @@ deps/vscode/*
 
 # Qdrant
 qdrant_storage/
+
+# allow multiple local clones with different workspaces with different colors
+# to make it easier to work on features in parallel
+*.code-workspace
```

Comment:
unrelated, but i like to work with the same repo open multiple times, and this is an easy way to do that, no functional change as there was no such file in the repo